xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/XWikiAction.java

```diff
@@ -85,6 +85,8 @@
 import com.xpn.xwiki.objects.BaseObject;
 import com.xpn.xwiki.plugin.fileupload.FileUploadPlugin;
 
+import net.sf.json.JSONObject;
```

Comment:
You might want to avoid using this very old abandoned library and use Jackson instead for example.

Reply:
OK I'll change that.